Sun, 01 Apr 2018 17:44:04 GMT node-telegram-bot-api deprecated Automatic enabling of cancellation of promises is deprecated.
In the future, you will have to enable it yourself.
See https://github.com/yagop/node-telegram-bot-api/issues/319. at module.js:641:30
Sun, 01 Apr 2018 18:32:24 GMT node-telegram-bot-api deprecated Automatic enabling of cancellation of promises is deprecated.
In the future, you will have to enable it yourself.
See https://github.com/yagop/node-telegram-bot-api/issues/319. at module.js:641:30
You have triggered an unhandledRejection, you may have forgotten to catch a Promise rejection:
MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (/Users/hazratgs/www/instalator-telegram/node_modules/mongoose/node_modules/mongodb-core/lib/topologies/server.js:503:11)
    at emitOne (events.js:125:13)
    at Pool.emit (events.js:221:7)
    at Connection.<anonymous> (/Users/hazratgs/www/instalator-telegram/node_modules/mongoose/node_modules/mongodb-core/lib/connection/pool.js:326:12)
    at Object.onceWrapper (events.js:326:30)
    at emitTwo (events.js:135:13)
    at Connection.emit (events.js:224:7)
    at Socket.<anonymous> (/Users/hazratgs/www/instalator-telegram/node_modules/mongoose/node_modules/mongodb-core/lib/connection/connection.js:245:50)
    at Object.onceWrapper (events.js:324:30)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
